A47: xDS Least Request LB Policy

erikjoh opened the pull request.

fdfzcq commented on A47-xds-least-request-lb-policy.md:
```diff
@@ -0,0 +1,181 @@
+field](https://github.com/envoyproxy/envoy/blob/2443032526cf6e50d63d35770df9473dd0460fc0/api/envoy/config/cluster/v3/cluster.proto#L722).
+Prior to this proposal, gRPC supported two values for this field,
+`ROUND_ROBIN` and `RING_HASH`.  With this proposal, we will add support for an
+additional policy, `LEAST_REQUEST`.
```
Should this support a list of load balancing algorithms as what google suggested in the previous meeting?

erikjoh replied:
Supporting a list of lb policies is a much larger effort that probably will get a gRFC of its own :sweat_smile:

erikjoh committed:
A48: xDS Least Request LB Policy

erikjoh committed:
Update approver

erikjoh committed:
Add missing semicolon

erikjoh committed:
Address initial review and remove TODOs

erikjoh committed:
Add that others also may contribute to implementation

erikjoh committed:
Adress PR comments

erikjoh committed:
Remove "_XDS" from environment variable

Realized that this LB policy isn't really tied to xDS.
Therefore, the environment variable doesn't need to contain "XDS".
